python/tests-unit/test_crypto.py

```diff
@@ -7,6 +7,7 @@
 from unittest import mock
 
 from insights_ansible_playbook_lib import crypto
+from insights_ansible_playbook_lib import _keygen
 
 
 # GPG2 exports keys in a way gpg1.4 cannot read. Since gpg1.4 is present in the
```

Comment:
With keys not being hardcoded anymore, we can drop this comment completely.

Reply:
Resolved